pkg/odo/cli/service/service.go

```diff
@@ -5,208 +5,33 @@ import (
-	Run: func(cmd *cobra.Command, args []string) {
-		glog.V(4).Infof("service delete called\n args: %#v", strings.Join(args, " "))
-
-		context := genericclioptions.NewContext(cmd)
-		client := context.Client
-		applicationName := context.Application
-
-		serviceName := args[0]
-
-		// Checks to see if the service actually exists
-		exists, err := svc.SvcExists(client, serviceName, applicationName)
-		util.CheckError(err, "unable to delete service because Service Catalog is not enabled in your cluster")
-		if !exists {
-			fmt.Printf("Service with the name %s does not exist in the current application\n", serviceName)
-			os.Exit(1)
-		}
-
-		var confirmDeletion string
-		if serviceForceDeleteFlag {
-			confirmDeletion = "y"
-		} else {
-			fmt.Printf("Are you sure you want to delete %v from %v? [y/N] ", serviceName, applicationName)
-			fmt.Scanln(&confirmDeletion)
-		}
-
-		if strings.ToLower(confirmDeletion) == "y" {
-			err := svc.DeleteService(client, serviceName, applicationName)
-			util.CheckError(err, "")
-			fmt.Printf("Service %s from application %s has been deleted\n", serviceName, applicationName)
-
-		} else {
-			fmt.Printf("Aborting deletion of service: %v\n", serviceName)
-		}
-	},
-}
-
-var serviceListCmd = &cobra.Command{
-	Use:   "list",
-	Short: "List all services in the current application",
-	Long:  "List all services in the current application",
-	Example: `  # List all services in the application
-  odo service list
-	`,
-	Args: cobra.NoArgs,
-	Run: func(cmd *cobra.Command, args []string) {
-		context := genericclioptions.NewContext(cmd)
-		client := context.Client
-		applicationName := context.Application
-
-		services, err := svc.ListWithDetailedStatus(client, applicationName)
-		util.CheckError(err, "Service Catalog is not enabled in your cluster")
-
-		if len(services) == 0 {
-			fmt.Println("There are no services deployed for this application")
-			return
-		}
-
-		w := tabwriter.NewWriter(os.Stdout, 5, 2, 3, ' ', tabwriter.TabIndent)
-		fmt.Fprintln(w, "NAME", "\t", "TYPE", "\t", "STATUS")
-		for _, comp := range services {
-			fmt.Fprintln(w, comp.Name, "\t", comp.Type, "\t", comp.Status)
-		}
-		w.Flush()
-	},
-}
-
-func NewCmdService() *cobra.Command {
-	serviceDeleteCmd.Flags().BoolVarP(&serviceForceDeleteFlag, "force", "f", false, "Delete service without prompting")
-	serviceCreateCmd.Flags().StringVar(&plan, "plan", "", "The name of the plan of the service to be created")
-	serviceCreateCmd.Flags().StringSliceVarP(&parameters, "parameters", "p", []string{}, "Parameters of the plan where a parameter is expressed as <key>=<value")
-
-	// Add a defined annotation in order to appear in the help menu
-	serviceCmd.Annotations = map[string]string{"command": "other"}
+// RecommendedCommandName is the recommended service command name
+const RecommendedCommandName = "service"
+
+var serviceLongDesc = ktemplates.LongDesc(`
+Perform service catalog operations, Limited to template service broker only.`)
```

Comment:
Although this is a problem with the PR itself, I nonetheless think this PR provides a change to fix this erroneous statement - the service operations also work with the Ansible Service Broker

Reply:
@geoand Not sure I understand what you mean? What I think you meant is: although this is *not* a problem with the PR itself, this PR provides an *opportunity* to address this erroneous statement. Correct?

Reply:
Damn it... I'm really sorry about this... 
I know how frustrating statements like mine above can be, especially when they occur on a regular basis... 
I really need to make a serious effort to be align my hands with my mind when I type!!!

You are absolutely correct with your interpretation of my (very badly written) statement

Reply:
Hehe, no worries just wanted to make sure I understood your concern properly.

Reply:
Fixed.

Reply:
Thanks!